orangecontrib/spectroscopy/agilent.py

```diff
@@ -245,62 +353,186 @@ class agilentMosaic(DataObject):
     https://bitbucket.org/AlexHenderson/agilent-file-formats
     """
 
+    def __init__(self, filename, MAT=False):
+        super().__init__(filename, MAT)
+        self._get_data()
+
+    def _get_data(self):
```

Comment:
For, the name _get_data was a bit confusing and was looking for what it actually returns. Perhaps a name such as _prepare_data or _load_data or similar would be more appropriate next time.